<?xml version="1.0"?>
<launch>
  <!-- Lidar with mission node -->
  <rosparam command="load" file="$(find new_lane)/cfg/frenet_planner.yaml"/>
  <node pkg="new_lane" name="lane_changer_node" type="lane_changer_node" output="screen">
  	
  	<!--param for clustering-->
  	<param name="cluster_tolerance"       value="1.0"/>
  	<param name="cluster_minsize"         value="4"/>
  	<param name="cluster_maxsize"         value="300"/>
  	
  	<!--param for static-->
  	<!-- <param name="x_min_st"                 value="-0.7"/>
  	<param name="x_max_st"                 value="1.2"/>
  	<param name="y_min_st"                 value="-0.7"/>
  	<param name="y_max_st"                 value="0.7"/>  
	<param name="z_min_st"                 value="-"/>
  	<param name="z_max_st"                 value="0.0"/>  	 -->

  	<param name="x_min_st"                 value="0"/>
  	<param name="x_max_st"                 value="30"/>
  	<param name="y_min_st"                 value="-10"/>
  	<param name="y_max_st"                 value="10"/>  
	<param name="z_min_st"                 value="-1.6"/>
  	<param name="z_max_st"                 value="2.0"/>  	
  	
   	<!--param for whole misstion-->  	
  	<param name="str_"       value="0.6"/>
  	<param name="sepe"       value="1000"/>
 </node>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find new_lane)/rviz/lane_changer_rviz.rviz"/>
</launch>
